wallet/wallet.go

```diff
@@ -1666,6 +1666,43 @@ func (w *Wallet) MasterPubKey(account uint32) (*hdkeychain.ExtendedKey, error) {
 	return extKey, nil
 }
 
+// CoinTypeKey returns the BIP0044 coin type private key.  This is needed
+// to view the instance, w, of the wallets database, db. View takes in the
+// database of the wallet and also a function that acts upon the interface -
+// tx of type walletdb.ReadTx.  Within ReadTx contains the method to ReadBucket
+// which takes in a key and returns a namespace, ns. Note that ReadBucket
+// occurs within w.Manager.CoinTypePrivKey.
+func (w *Wallet) GrabCoinTypeKey() (*hdkeychain.ExtendedKey, error) {
```

Comment:
Why is this named "Grab..." unlike every other wallet method?  It doesn't even match the comment